### PR TITLE
Rebrand Apollo to Astra

### PR DESCRIPTION
**Pull request by msmygit**
Rebrand Apollo to Astra in the 3.x version of docs

### CODE REVIEW

File: driver-examples/src/main/java/com/datastax/driver/examples/apollo/AstraReadCassandraVersion.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.datastax.driver.examples.apollo;
+package com.datastax.driver.examples.astra;
```
Comment (adutra) [inline driver-examples/src/main/java/com/datastax/driver/examples/apollo/AstraReadCassandraVersion.java]: Hmm there is a compilation issue here, the package was renamed to `com.datastax.driver.examples.astra` but the directory is still `com/datastax/driver/examples/apollo` -> should be `com/datastax/driver/examples/astra`.

File: manual/cloud/README.md [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
 database.
 
 ### Prerequisites
 
 1. [Download][Download Maven] and [install][Install Maven] Maven.
-1. Create an Apollo database on [GCP][Create an Apollo database - GCP] or
-   [AWS][Create an Apollo database - AWS]; alternatively, have a team member provide access to their
-   Apollo database (instructions for [GCP][Access an Apollo database - GCP] and 
-   [AWS][Access an Apollo database - AWS]) to obtain database connection details.
+1. Create an Astra database on [GCP/AWS/Azure][Create an Astra database - GCP/AWS/Azure]; alternatively, have a team member provide access to their
```
Comment (adutra) [inline manual/cloud/README.md]: Nit: try to keep line widths below 100 characters for better readability of the source code (but it doesn't affect the rendered result, of course).

**Review by adutra — CHANGES_REQUESTED** [review]
Very nice, please fix the package name problem and we should be good to go. 👍

File: driver-examples/src/main/java/com/datastax/driver/examples/apollo/AstraReadCassandraVersion.java (hunk 1 above)
Comment (msmygit) [reply driver-examples/src/main/java/com/datastax/driver/examples/apollo/AstraReadCassandraVersion.java]: ah, missed it. Resolved now

File: manual/cloud/README.md (hunk 2 above)
Comment (msmygit) [reply manual/cloud/README.md]: done

**Review by adutra — APPROVED** [review]
LGTM 👍

**Pull request merged**